### PR TITLE
tools: Link checker warns about redirections

### PR DESCRIPTION
**Pull request by marusak**

### CODE REVIEW

**Comment by marusak** [comment]
Heh, needs tests adjusting

File: tools/urls-check [hunk 1]
```diff
@@ -35,6 +35,9 @@ sys.dont_write_bytecode = True
 DAYS = 7
 TASK_NAME = "Validate all URLs"
 
+known_redirects = ["https://access.redhat.com/security/updates/classification/#",
+                   "https://firefox.com/", "https://www.microsoft.com/"]
```
Comment (martinpitt) [inline tools/urls-check:39]: Aren't these obsolete with the previous commit?

Comment (marusak) [reply tools/urls-check:39]: No they are not.
The first one is in condition, the other twos change to localized page, so I don't want to force there `en_us` or so.

**Review by martinpitt — APPROVED** [review]
Thanks! Looks good, assuming green tests.

**Pull request merged**